src/Commands/BackupCommand.php

```diff
@@ -59,6 +69,15 @@ public function handle()
 
             consoleOutput()->comment('Backup completed!');
         } catch (Exception $exception) {
+            if ($this->tries > 1 && $this->currentTry < $this->tries) {
+                if (!empty(config('backup.backup.retry_delay'))) {
+                    sleep((int)config('backup.backup.retry_delay'));
```

Comment:
Instead of using `sleep` use the new sleep helper that is easier to test: https://www.amitmerchant.com/the-new-sleep-helper-in-laravel-10/